README.md

```diff
@@ -120,7 +120,27 @@ $ run-full-environment
 ```
 
 It is also possible to only run Nginx in that shell with `run-nginx`. This is
-usefull for instance to proxy a `cty serve` running from GHCi.
+useful for instance to proxy a `cty serve` running from GHCi.
+
+The `run-vm-test` attribute let's you run a NixOS VM integration test
+that mirrors the production setup. You can run it either interactively
+or non-interactively:
+
+```sh
+$ nix-build -A run-vm-test
+# ^ Will run the VM test in a non interactive fashion.
+$ nix-build -A run-vm-test.driverInteractive && ./result/bin/nixos-test-driver
+# ^ Will open two qemu windows, one for the server, one for the client.
```

Comment:
Is there a missing instruction here, like typing `start_all()` or something ?

Reply:
We do! Thanks for the heads up.